### DISCUSSION TITLE
How to get params trough DBUS

### DISCUSSION THREAD

**Discussion by Cacsjep** [AxisCommunications/acap-native-sdk-examples#217] [General]
When utilizing the axparameter example to access Properties.System.SerialNumber, it works but logs an error in the package log. I haven't discovered any documentation on retrieving parameters through DBUS. Should the approach instead involve obtaining VAPIX credentials via DBUS and then using param.cgi to retrieve the parameters? 

`Vegafied parameter used through libparhand (Client:./axparamexample Parameter:root.Properties.System.SerialNumber) Please use it through its DBus service`

**Comment by pataxis** [comment]
Hi @Cacsjep , thanks for your question.

No it should not be necessary and we don't see this in tests.
Have you modified the axparameter example?

Please also state Product, AXIS OS version and ACAP Native SDK version.
